.slug-post-clean

```diff
@@ -0,0 +1,5 @@
+vendor/bundle/ruby/?.?.?/gems/wkhtmltopdf-binary-*/bin/wkhtmltopdf_macos_cocoa.gz
```

Comment:
I see now that this will fail in case one of the Ruby version numbers is major than 9. For example: `vendor/bundle/ruby/3.1.20/gems...`

Can you try modifying the experssions replacing `?` for `*` in the Ruby version:
`vendor/bundle/ruby/*.*.*/gems/wkhtmltopdf-binary-*/bin/wkhtmltopdf_macos_cocoa.gz`

Reply:
Thanks, let's see if this still works